cmds/task/actors.go

```diff
@@ -5,42 +5,74 @@ import (
 	"io"
 	"os"
 	"time"
-
+	
 	"github.com/spf13/pflag"
 	"github.com/taskcluster/slugid-go/slugid"
 	tcclient "github.com/taskcluster/taskcluster-client-go"
 	"github.com/taskcluster/taskcluster-client-go/queue"
 )
 
 // runCancel cancels the runs of a given task.
-func runCancel(credentials *tcclient.Credentials, args []string, out io.Writer, _ *pflag.FlagSet) error {
+func runCancel(credentials *tcclient.Credentials, args []string, out io.Writer, flagSet *pflag.FlagSet) error {
+	nooprunCancel, _ := flagSet.GetBool("noop")
+
 	q := makeQueue(credentials)
 	taskID := args[0]
 
+
+	if nooprunCancel {
+		c, _ := q.Status(taskID)
+		run := c.Status.Runs[len(c.Status.Runs)-1]
+		fmt.Println("Cancels", taskID ,"(state:",run.State,")")
+		runName(credentials , args , out , flagSet)
```

Comment:
It's not clear why this is a separate function call. Should we have the task name in the `Println` above? Having all the information on one line should help us if we run the `taskcluster` command inside scripts, or want to `grep` the output. 

There are options here:
1. We could take the contents of `runName` and move them into this function instead, giving us the name.
2. We could make a different version of `runName` that uses `return` to give us the name, which we can then use.
3. We could make a new function that prints out a `noop` message, and uses one of option 1 or 2. This lets us re-use the same code over and over for the no-op messages, and only change the format of them in one place.

For option 3, imagine we have:

    if nooprunCancel {
        displayNoopMsg(taskID, "Cancels", out)
    }

`func displayNoopMsg` can then look up all the information and have `fmt.Println(actionMsg, taskID, ...` 

What do you think?

Reply:
Sounds awesome. Will try option 1. and 2. as soon as possible.